swagger_server/controllers/itm_ta2_eval_controller.py

```diff
@@ -149,6 +151,7 @@ def start_session(adm_name, session_type, kdma_training=None, max_scenarios=None
         adm_name=adm_name,
         session_type=session_type,
         kdma_training=kdma_training,
+        adm_profile=adm_profile if adm_profile != 'None' else None,
```

Comment:
I haven't tested yet, but just looking at the code in server and client, I am curious how adm_profile could ever be the string "None". I see it as an empty string and as the value None. Is there some place I'm missing where it is a string None, and that will always be passed in if it's an empty string or the value None?

Reply:
It's been a while since I added this / was testing this, but I seem to recall that when I was using the `itm_minimal_runner` client and did not supply an `adm_profile` parameter on the command line, that it would get propagated to the TA2 controller as the string 'None'.  I believe this is because the client always sends the `adm_profile` parameter, even if it wasn't on the command line.  Presumably it sends `None` across the wire, but the server code interprets it as the string "None".

You can test this by using the `itm_minimal_runner` client, don't set `adm_profile`, and see what the TA2 controller gets.  I *believe* it will get the string 'None', hence the extra code.

Another approach could have been to change the client only to send the `adm_profile` if a value was set.  But adding the code to the server accounts for if the client doesn't do this, which it arguably should.

Reply:
That makes sense. I was wondering if it was some conversion "problem" from client to server.